prow/test/integration/config/prow/cluster/gangway.yaml

```diff
@@ -0,0 +1,132 @@
+metadata:
+  name: gangway
+  namespace: default
+  labels:
+    app: gangway
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gangway
+  template:
+    metadata:
+      labels:
+        app: gangway
+    spec:
+      terminationGracePeriodSeconds: 1
+      serviceAccountName: gangway
+      containers:
+      - name: gangway
+        image: localhost:5001/gangway
+        args:
+        - --config-path=/etc/config/config.yaml
+        - --job-config-path=/etc/job-config
+        - --grace-period=110s
+        # This cookie file is only here to trigger the creation of a
+        # Gerrit-flavored Git client factory. So this makes this gangway deployment
+        # "tied" to Gerrit.
+        #
+        # TODO (listx): Allow gangway to be deployed with access to multiple
+        # GitHub/Gerrit credentials (and make it know which one to use based on
+        # the org/repo name). We can't simply deploy a 2nd gangway deployment
+        # configured with GitHub creds to test that codepath because currently
+        # gangway will always choose one or the other.
+        - --cookiefile=/etc/cookies/cookies
+        - --dry-run=false
+        ports:
+        - name: grpc
+          containerPort: 32000
+        - name: http
+          containerPort: 80
+        - name: metrics
+          containerPort: 9090
+        volumeMounts:
+        - name: cookies
+          mountPath: /etc/cookies
+          readOnly: true
+        - name: config
+          mountPath: /etc/config
+          readOnly: true
+        - name: job-config
+          mountPath: /etc/job-config
+          readOnly: true
+        resources:
+          requests:
+            cpu: "1"
+            memory: "2Gi"
+        env:
+        # When cloning from an inrepoconfig repo, don't bother verifying certs.
+        # This allows us to use "https://..." addresses to fakegitserver.
+        - name: GIT_SSL_NO_VERIFY
+          value: "1"
+      volumes:
+      - name: cookies
+        secret:
+          defaultMode: 420
+          secretName: http-cookiefile
+      - name: config
+        configMap:
+          name: config
+      - name: job-config
+        configMap:
+          name: job-config
+      tolerations:
+      - key: "prowcomponent"
+        operator: "Equal"
+        value: "true"
+        effect: "NoSchedule"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: gangway
+  namespace: default
+spec:
+  selector:
+    app: gangway
+  ports:
+  - port: 32000
+    targetPort: 32000
+    nodePort: 32000
+    protocol: TCP
+    name: grpc
+  - port: 80
+    targetPort: 80
+    protocol: TCP
+    name: http
+  type: NodePort
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: gangway
+  namespace: default
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gangway
+rules:
+  - apiGroups:
+      - prow.k8s.io
+    resources:
+      - prowjobs
+    verbs:
+      - create
+      - get
+      - list
+      - update
```

Comment:
Is it correct that these verbs are matched to the [methods](https://docs.google.com/document/d/1v77jp1Nb5C2C2-PdV02SGViO9CyZ9SvNxCPOHyIUQeo/edit#heading=h.13cp7vwywar6) (`create`, `get`, `list`)?
If yes, then the verb `update` matches to no method.. 🤔

Reply:
Ack. This is more of a result of me copy-pasting some settings from another component.

But in theory the 1:1 correspondence is more of a coincidence than intentional design. The design currently does not advocate exposing Prow Job CRs directly over the API. Instead clients are supposed to use an external-facing, separate data layer for job executions. It's just that currently the executions map the Prow Job CRs 1:1 because it's the most convenient thing to do (without having to overhaul the rest of the Prow components).

Reply:
Removed `update` verb as it is not used.